guides/v2.2/frontend-dev-guide/translations/xlate.md

```diff
@@ -164,6 +164,8 @@ If you need help understanding the context or meaning of a UI string, or have qu
 * [Use translation dictionary to customize strings]
 * [Translate theme strings]
 * [Example theme translation dictionary]
+* [Magento translations GitHub project]
+
```

Comment:
```suggestion

```

Reply:
Hi @diazwatson
Thanks for your contributions. Can you please remove the extra line.